### PR TITLE
support SCRIPTSTORE and MODELSTORE

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

File: redisai/commands_test.go [hunk 1]
```diff
@@ -467,6 +467,50 @@ func TestCommand_ModelSet(t *testing.T) {
...
+		data            []byte
+		inputs          []string
+		outputs         []string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{keyModelStore1, args{keyModelStore1, BackendTF, DeviceCPU, "", 0, 0, 0, data, []string{"transaction", "reference"}, []string{"output"}}, false},
```
Comment (alonre24) [inline redisai/commands_test.go:499]: Please add a short documentation for every test case

File: redisai/commands_test.go [hunk 2]
```diff
@@ -889,6 +933,65 @@ func TestCommand_ScriptSet(t *testing.T) {
...
+		device      string
+		data        string
+		entryPoints []string
+		tag         string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
```
Comment (alonre24) [inline redisai/commands_test.go:952]: Please add a short documentation for every test case

File: redisai/commands.go [hunk 3]
```diff
@@ -143,26 +157,47 @@ func (c *Client) ModelRun(name string, inputTensorNames, outputTensorNames []str
...
+
+// ScriptStoreWithTag store a TorchScript as the value of a key with tag.
+func (c *Client) ScriptStoreWithTag(name, device, scriptSource string, entryPoints []string, tag string) (err error) {
+	args := scriptStoreFlatArgs(name, device, tag, entryPoints, scriptSource)
+	_, err = c.DoOrSend("AI.SCRIPTSTORE", args, nil)
+	return
+}
+
+// ScriptStoreFromInteface store a TorchScript as the value from a structure that implements the ScriptInterface
+func (c *Client) ScriptStoreFromInteface(keyName string, script ScriptInterface) (err error) {
```
Comment (alonre24) [inline redisai/commands.go:195]: typo `Inteface`

File: redisai/commands_test.go (hunk 1 above)
Comment (AvitalFineRedis) [reply redisai/commands_test.go:499]: It's hard to know here :sweat_smile:
I copied this from the `AI.MODELSET` test

**Review by alonre24 — APPROVED** [review]

**Pull request merged**